src/osd/osd_types.h

```diff
@@ -5925,6 +5926,27 @@ struct pool_pg_num_history_t {
 };
 WRITE_CLASS_ENCODER(pool_pg_num_history_t)
 
+// prefix pgmeta_oid keys with _ so that PGLog::read_log_and_missing() can
+// easily skip them
+static const string_view infover_key = "_infover"sv;
```

Comment:
what's the benefit of using string_view when we always convert these to string for storing in a container? is the temporary somehow elided in the encode() case?

Reply:
Crimson uses string_view a lot internally.  I think it's a wash for classic, but it's convenient for crimson.